extensions/spark/kyuubi-spark-listener/src/main/scala/org/apache/kyuubi/plugin/lineage/events/OperationLineageEvent.scala

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.plugin.lineage.events
+
+import org.apache.kyuubi.Utils
+import org.apache.kyuubi.events.KyuubiEvent
+
+case class Lineage(
```

Comment:
can we add some comments to list some examples of the `Lineage`, it may be used by developers.

Reply:
There will also be detailed instructions in the [docs/extensions/engines/spark/lineage.md](https://github.com/apache/incubator-kyuubi/pull/3185/files/e1abba76741e31bf66897532da43530516618173#diff-815ac878f18bdecd045087459e56c3d38eed6a967f8a2d300bb43aefd9e8a679)